app/models/exercise/approach.rb

```diff
@@ -27,6 +27,7 @@ class Exercise::Approach < ApplicationRecord
   has_one :track, through: :exercise
 
   scope :random, -> { order('RAND()') }
+  scope :sorted, -> { order(:position) }
```

Comment:
I'd probably set this to a "default scope" instead of having to remember to use this each time.